test/bulleted_list_2_space_start_indent.json

```diff
@@ -0,0 +1,7 @@
+{
+    "default": true,
+    "MD007": {
+        "start_indented": true,
+        "start_indent": 2
```

Comment:
Love that you added test cases, by the way - could you add another where `start_indent` is 0?